vue-thacer/src/components/TheCeramicImages.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-  <div v-if="imageUrlArrayList !== null">
+  <div v-if="imageUrlArrayList.PHOTO.length != 0">
```

Comment:
Ah, try to use `==!`
or just `>`

Another thing, are you sure that PHOTO always exist and is an array ? If not there could be type error. maybe `imageUrlArrayList.PHOTO?.length`

Reply:
Ah it's here, trying to make sense of the `==` column  : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#comparing_equality_methods   🤯😱

Reply:
first one seems to work.
second, yes PHOTO always exist
Thanks